.github/workflows/docs.yaml

```
@@ -37,6 +37,19 @@ jobs:
               name: DocumentationHTML
               path: docs/_build/ape
 
+        - name: Doctesting
+          run: sphinx-build -b doctest docs docs/_build/doctest
+        
+        - name: Check doctest results
+          run: |
```

Think you need to combine these steps for it to work
```suggestion
          run: |
            sphinx-build -b doctest docs docs/_build/doctest
```